pkg/daemon/update.go

```diff
@@ -180,6 +181,11 @@ func (dn *Daemon) reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) error
 	newIgn := newConfig.Spec.Config
 
 	// Ignition section
+	// First check if this is a generally valid Ignition Config
+	rpt := validate.ValidateWithoutSource(reflect.ValueOf(newIgn))
+	if rpt.IsFatal() {
+		return errors.Errorf("Invalid Ignition config found: %v", rpt)
```

Comment:
Lowercase error

Reply:
so you are saying `rpt, err := validate`...?

Reply:
Nope, just the format of the error message :) (the actual sentence in the error, start it lowercase)

Reply:
Gotcha! I'll update the whole file this way to be consistent in a separate PR! :smile: